Projects/App/Sources/Presentation/MainHome/MainHomeViewController.swift

```diff
@@ -0,0 +1,317 @@
+//
+//  MainHomeViewController.swift
+//  App
+//
+//  Created by kokojong on 2023/02/12.
+//  Copyright © 2023 com.promise8. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+import RxCocoa
+import RxSwift
+
+final class MainHomeViewController: UIViewController {
+    
+    // MARK: - Properties
+    private let bag = DisposeBag()
+    var viewModel: MainHomeViewModel?
+    enum Section: CaseIterable {
+        case proceeding
+        case ended
+    }
+    var isOneStepPaging = true
+    var currentIndex: CGFloat = 0 {
+        didSet {
+            pageLabel.customAttributedString(text: "\(Int(currentIndex)+1)/5", highlightText: "\(Int(currentIndex)+1)", textFont: .www(size: 14)!, highlightTextFont: .www(size: 14)!, textColor: .wwwColor(.Gray450), highlightTextColor: .wwwColor(.WWWGreen))
+        }
+    }
+    
+    var fetchedMainHomeMeeting: MainHomeMeeting = .init(proceedingMeetings: [], endedMeetings: [])
+    
```

Comment:
p1) 접근지정자를 붙여주는건어떄요?  🔐
```suggestion
    
   private  var fetchedMainHomeMeeting: MainHomeMeeting = .init(proceedingMeetings: [], endedMeetings: [])
    
```

Reply:
@chaneeii 
엇 놓쳤던 부분이다! 확인할게!

Reply:
0fc77f9198a1e61a62237ae71be159ee79e6ecdf 에 반영